.github/workflows/deploy.yaml

```diff
@@ -386,13 +392,37 @@ jobs:
           deployment/seed-data.sh
         working-directory: .
 
+      - uses: actions/download-artifact@v3
+        with:
+          name: gaiaxkey
+          path: deployment/terraform/participant
+
+      - name: 'Push region verifiable credential'
+        run: |
+          mvn dependency:copy -Dartifact=org.eclipse.dataspaceconnector.identityhub:identity-hub-cli:$cli_version:jar:all -DoutputDirectory=.
+          
+          id=$(uuidgen)
+          
+          java -jar identity-hub-cli-$cli_version-all.jar -s=$server vc add \
+            -c='{"id":"'$id'","credentialSubject":{"region":"'$region'"}}' \
+            -b="did:web:$DID_HOST" \
+            -i="did:web:$issuer" \
+            -k="gaiaxkey.pem"
+          
+          java -jar identity-hub-cli-$cli_version-all.jar -s=$server vc list
+        env:
+          cli_version: 0.0.1-SNAPSHOT
+          region: ${{ matrix.region }}
+          issuer: ${{ needs.Deploy-Dataspace.outputs.gaiax_did_host }}
+          server: http://${{ env.EDC_HOST }}:8181/api/identity-hub
```

Comment:
nit: rather than server we can say hub-url or identity-hub-url. Suggesting this as identity hub cli also usage the same wording for option -s.

But up to you to decide, if you prefer to keep it this way then we can keep as it is.